torch_geometric/datasets/mesh_cnn_dataloader.py

```diff
@@ -0,0 +1,97 @@
+import torch
+from os import mkdir
+import os.path as osp
+from torch_geometric.data.dataloader import DataLoader
+from torch_geometric.datasets.mesh_cnn_base_datasets import MeshCnnBaseDataset
+from torch_geometric.data.meshcnnhandler import MeshCNNHandler
+
+
+class MeshDataLoader:
```

Comment:
What prevents us from directly using the `torch_geometric.data.DataLoader` (as you already do in `meshcnn_faust.py`)?

Reply:
I am using different __iter__ function. Do you have a suggestion how to avoid creating this class?